lib/open_project/scm/adapters/local_client.rb

```diff
@@ -222,12 +222,53 @@ def scm_encode(to, from, str)
         # This is compatible, but quite inefficient, so should
         # be run asynchronously.
         def count_required_storage
-          bytes = 0
-          ::Find.find(local_repository_path) do |f|
-            bytes += File.size(f) if File.file?(f)
+          count_storage_du.presence || count_storage_fallback
```

Comment:
`count_storage_du` can only return `nil` or an `Integer`. So the `presence` call is superfluous, isn't it?